src/mon/MonCommands.h

```diff
@@ -712,6 +712,13 @@ COMMAND("osd create " \
 	"name=uuid,type=CephUUID,req=false " \
 	"name=id,type=CephOsdName,req=false", \
 	"create new osd (with optional UUID and ID)", "osd", "rw", "cli,rest")
+COMMAND("osd new " \
+        "name=replace_id,type=CephOsdName,req=true " \
```

Comment:
I think the id should be optional? so that osd new will also work to create a fresh osd?  That probably means switching the argument order around